src/main/java/redis/clients/jedis/Connection.java

```diff
@@ -67,9 +70,43 @@ public Connection(final JedisSocketFactory socketFactory, JedisClientConfig clie
     initializeFromClientConfig(clientConfig);
   }
 
-  @Override
-  public String toString() {
-    return "Connection{" + socketFactory + "}";
+  public String toIdentityString() {
```

Comment:
Keep the existing one.
```suggestion
  @Override
  public String toString() {
    return "Connection{" + socketFactory + "}";
  }

  public String toIdentityString() {
```

Reply:
Ohhh.... Sorry about it. My mistake.